### PR TITLE
Cookbook update and small fix when decoding event topics

### PR DESCRIPTION
**Pull request by popenta**

### CODE REVIEW

File: multiversx_sdk/core/transactions_outcome_parsers/resources.py [hunk 1]
```diff
@@ -60,7 +60,19 @@ def find_events_by_identifier(transaction_outcome: TransactionOutcome, identifie
...
-    return find_events_by_predicate(transaction_outcome, lambda event: event.topics[0].decode() == topic if len(event.topics) else False)
+    def is_topic_matching(event: TransactionEvent):
+        if not len(event.topics):
+            return False
+
+        try:
+            decoded_topic = event.topics[0].decode()
+            if decoded_topic == topic:
+                return True
+            return False
```
Comment (andreibancioiu) [inline multiversx_sdk/core/transactions_outcome_parsers/resources.py:71]: Can be simplified / one-liner.

File: multiversx_sdk/core/transactions_outcome_parsers/resources.py [hunk 2]
```diff
@@ -60,7 +60,19 @@ def find_events_by_identifier(transaction_outcome: TransactionOutcome, identifie
...
+        except:
```
Comment (andreibancioiu) [inline multiversx_sdk/core/transactions_outcome_parsers/resources.py:72]: Optionally, can nominate the specific exception e.g. `UnicodeDecodeError`.

File: multiversx_sdk/core/transactions_outcome_parsers/transaction_events_parser_test.py [hunk 3]
```diff
@@ -167,3 +169,35 @@ def test_parse_event_with_multi_values():
...
         c=42
     )
+
+
+@pytest.mark.networkInteraction
+def test_multisig_start_perform_action():
+    from multiversx_sdk import (ApiNetworkProvider, TransactionEventsParser,
+                                TransactionsConverter,
+                                find_events_by_first_topic)
+    from multiversx_sdk.abi import Abi
```
Comment (andreibancioiu) [inline multiversx_sdk/core/transactions_outcome_parsers/transaction_events_parser_test.py:179]: Not in the import section?

File: multiversx_sdk/core/transactions_outcome_parsers/transaction_events_parser_test.py [hunk 4]
```diff
@@ -167,3 +169,35 @@ def test_parse_event_with_multi_values():
...
+
+    api = ApiNetworkProvider("https://testnet-api.multiversx.com")
+    converter = TransactionsConverter()
+
+    transaction_on_network = api.get_transaction("6f006c99e45525c94629db2442d9ca27ff088ad113a09f0a3a3e24bcc164945a")
+    transaction_outcome = converter.transaction_on_network_to_outcome(transaction_on_network)
+
+    abi = Abi.load(Path(testdata / "multisig-full.abi.json"))
```
Comment (andreibancioiu) [inline multiversx_sdk/core/transactions_outcome_parsers/transaction_events_parser_test.py:187]: `testdata` is already a `Path`, can be directly `testdata / "multisig-full.abi.json"`.

File: multiversx_sdk/core/transactions_outcome_parsers/resources.py (hunk 1 above)
Comment (popenta) [reply multiversx_sdk/core/transactions_outcome_parsers/resources.py:71]: done

File: multiversx_sdk/core/transactions_outcome_parsers/resources.py (hunk 2 above)
Comment (popenta) [reply multiversx_sdk/core/transactions_outcome_parsers/resources.py:72]: done

File: multiversx_sdk/core/transactions_outcome_parsers/transaction_events_parser_test.py (hunk 4 above)
Comment (popenta) [reply multiversx_sdk/core/transactions_outcome_parsers/transaction_events_parser_test.py:187]: indeed, done

File: multiversx_sdk/core/transactions_outcome_parsers/transaction_events_parser_test.py (hunk 3 above)
Comment (popenta) [reply multiversx_sdk/core/transactions_outcome_parsers/transaction_events_parser_test.py:179]: forgot about them. moved in the import section

**Review by axenteoctavian — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**